sentry-rails/lib/sentry/rails/active_job.rb

```diff
@@ -79,6 +79,12 @@ def sentry_context(job)
 
           def sentry_serialize_arguments(argument)
             case argument
+            when Range
+              if argument.first.is_a?(ActiveSupport::TimeWithZone)
```

Comment:
This would still not support ranges of floats for example. (E.g. `(1.1...1.5)`) Is that intentional, just to keep this fix small?

Reply:
@solnic Also, beware that `Range#first` raises an exception if it's a beginless range (..1).

```irb
(..1).first
'Range#first': cannot get the first element of beginless range (RangeError)
```

Unlike `begin`, which would return nil.

Reply:
@maxim I just left original behavior untouched, but eventually I'd vote for changing it to always dump ranges to a string "{left}..{right}"

Reply:
> Also, beware that `Range#first` raises an exception if it's a beginless range (..1).

@maxim ah great catch, thanks I addressed it via 6d8f774c